runtime/caml/config.h

```diff
@@ -266,4 +266,15 @@ typedef uint64_t uintnat;
 /* Default size of runtime_events ringbuffers, in words, in powers of two */
 #define Default_runtime_events_log_wsize 16
 
+
+/* Size of cache line. Atomics allocated with aligned constructor are aligned
+   this value to avoid false sharing of cache line. */
+#ifdef TARGET_s390x
+   #define Cache_line_bsize 256
+#else
+   /* Many 64-bit processors have 128-byte cache line. Even with 64-byte cache
+   line, it may be beneficial to have twice that. */
+   #define Cache_line_bsize 128
+#endif
```

Comment:
Perhaps, a better name here is `Assumed_cache_line_bsize`?

Reply:
The comment clearly says that this is the assumed size of the cache line. So the name is fine.